cwms_radar_api/src/test/java/cwms/radar/data/dao/LocationGroupDaoTestIT.java

```diff
@@ -0,0 +1,59 @@
+package cwms.radar.data.dao;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import cwms.radar.api.DataApiTestIT;
+import cwms.radar.data.dto.LocationGroup;
+import fixtures.RadarApiSetupCallback;
+import java.util.List;
+import java.util.Optional;
+import mil.army.usace.hec.test.database.CwmsDatabaseContainer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("integration")
+class LocationGroupDaoTestIT extends DataApiTestIT {
+
+    @BeforeAll
+    public static void load_data() throws Exception {
+        loadSqlDataFromResource("cwms/radar/data/sql/store_loc_groups.sql");
+    }
+
+    @AfterAll
+    public static void deload_data() throws Exception {
+        loadSqlDataFromResource("cwms/radar/data/sql/store_loc_groups.sql");
```

Comment:
FYI, this is a good candidate for creating a template and function inside DataApiTestIT similar to createTS.

That way the locations can be registered and removed at the end of the test sequence. That's what allows for the tests to be cycled constantly without breaking. granted for this one not a huge deal as we can just keep calling create_location for the same name to know ill effect but it's a good practice to start getting into.